src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,120 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User firstUser;
+    private User secondUser;
+    private User thirdUser;
+    private User fourthUser;
+
+    @BeforeAll
+    public static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    public void setUp() {
+        firstUser = new User("Mike", "mike12345", 22);
+        secondUser = new User("Alice", "password", 19);
+        thirdUser = new User("Melisa", "qwerty", 20);
+        fourthUser = new User("MaX", "MaX24MaX", 24);
+    }
+
+    @Test
+    public void register_passwordLength_isValid() {

```

Comment:
Fix test mehod names according to readme
```suggestion
    public void register_passwordLength_ok() {

```